src/api/api.js

```diff
@@ -0,0 +1,37 @@
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/E0wnoGqC3httDh8BB8Bp/books';
+
+export const addBookToApi = async (book) => {
+  const newBook = await fetch(URL, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: book.id,
+      title: book.title,
+      category: book.category,
+    }),
+  });
+  return newBook;
+};
+
+export const removeBookFromApi = async (id) => {
+  const bookWillRemove = await fetch(`${URL}/${id}`, {
+    method: 'DELETE',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+    }),
+  });
+  return bookWillRemove;
+};
```

Comment:
- [ ] Delete method doesn't work as expected, it deletes the book but it doesn't send a response to the API. Take a look at the message displayed. 

- [ ] Please implement the remove button that is below not the remove button that appears after adding.

- [ ] If a book is deleted successfully, we need to see the message saying that a book has been deleted successfully.

![setup-8](https://user-images.githubusercontent.com/55635977/145471111-a16b1092-2914-4962-85c0-331efb9aad64.png)